### PR TITLE
8288601: Consolidate static/dynamic archive tables

### PR DESCRIPTION
**Pull request by iklam**
Please review this small clean-up of almost identical code that handles the tables for the static and dynamic archives.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8288601](https://bugs.openjdk.org/browse/JDK-8288601): Consolidate static/dynamic archive tables

### Reviewers
 * [Calvin Cheung](https://openjdk.org/census#ccheung) (@calvinccheung - **Reviewer**) ⚠️ Review applies to [1d15341b](https://git.openjdk.org/jdk/pull/9190/files/1d15341bf0ec495df98d60dd33547476ed9b337d)
 * @kristylee88 (no known github.com user name / role)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/9190/head:pull/9190` \
`$ git checkout pull/9190`

Update a local copy of the PR: \
`$ git checkout pull/9190` \
`$ git pull https://git.openjdk.org/jdk pull/9190/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 9190`

View PR using the GUI difftool: \
`$ git pr show -t 9190`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/9190.diff">https://git.openjdk.org/jdk/pull/9190.diff</a>

</details>

### CODE REVIEW

File: src/hotspot/share/cds/dumpTimeClassInfo.hpp [hunk 1]
```diff
@@ -24,6 +24,7 @@
 
 #ifndef SHARED_CDS_DUMPTIMESHAREDCLASSINFO_HPP
 #define SHARED_CDS_DUMPTIMESHAREDCLASSINFO_HPP
+
 #include "cds/archiveBuilder.hpp"
```
Comment (calvinccheung) [inline src/hotspot/share/cds/dumpTimeClassInfo.hpp:28]: Only a blank line was added.

**Review by calvinccheung — APPROVED** [review]
Nice cleanup.
Has this changeset gone through any tier(s) testing?

**Comment by iklam** [comment]
> Nice cleanup. Has this changeset gone through any tier(s) testing?

Hi Calvin,

Thanks for the review. This changeset has passed tiers 1-4. I also removed the blank line change you found.

**Review by kristylee88 — APPROVED** [review]

**Pull request closed without merging**